gatsby-config.js

```diff
@@ -423,6 +409,20 @@ module.exports = {
                             }
                         ]
                     },
+                    {
```

Comment:
Why it is under Cloud extension section, not under Edge section?

Reply:
The other is not edge section, it is the foundation section where we have all the setup and core requirements. All the other extensions are here.